scripts/stats/mesoscale/exevs_mesoscale_nam_grid2obs_stats.sh

```diff
@@ -362,7 +362,10 @@ echo "*****************************"
         MODEL_DIR=$(echo ${MODEL_DIR_PATH##*/})
         mkdir -p $COMOUT/$MODEL_DIR
         for FILE in $MODEL_DIR_PATH/*; do
-           cpreq -v $FILE $COMOUT/$MODEL_DIR/.
+           if [ -s $FILE ]; then
+              cp -v $FILE $COMOUT/$MODEL_DIR/.
+	   fi
+           # cpreq -v $FILE $COMOUT/$MODEL_DIR/.
```

Comment:
@RoshanShrestha-NOAA Can you please remove this commented out "cpreq" line? Thank you!
`# cpreq -v $FILE $COMOUT/$MODEL_DIR/.`

Reply:
In fact, if you could remove all commented out "cpreq" lines from this PR (I see several now), that would be great. Thank you!

@RoshanShrestha-NOAA @PerryShafran-NOAA

Reply:
I will remove them asap